### ISSUE TITLE
[BUG] Missing feedback in some forms

### ISSUE THREAD

**Issue by melcarthus** [BoltAuth/Auth#23]
-----------

When I edit my profile and submit the form I get no feedback. When I want to recover my password and I use an e-mailadres that's not registered I get no feedback. The `feedback template` is included inside the files. The only form that gives feedback is the `login form`.

The documentation is missing/lacking in explaining if there are twig-functions for this to do this manually.

Details
-------

 - Relevant Bolt Auth Version: [ 3.0.2 ]
 - Relevant Bolt Version: [ 3.4.10 ]
 - PHP version: [ 7.0 ]
 - For UX/UI issues: [ N/A ]

Reproduction
------------

Copy all needed files to the theme directory and set the paths accordingly inside the auth config.
Test the login / recovery / edit templates. You wil notice that only the login template gives feedback of being successfully logged in or out.

I looked in the source of my HTML and there is also no empty html tag from the feedback template. It does not show at all.

**Comment by SvanteRichter** [comment]
So this only happens when you use custom templates? (just trying to understand the issue better)

**Comment by melcarthus** [comment]
Yes happens in my custom templates, but my feedback template was unaltered at the time.

**Comment by SvanteRichter** [comment]
Could you perhaps walk me through all the configs and templates that you are using that are non-standard? Or even easier zip them up (with access tokens removed of course) and send them to me so I can have a look?

**Comment by alexlakedml** [comment]
> Yes happens in my custom templates, but my feedback template was unaltered at the time.
Was there ever a resolution to this problem? We're also seeing it.

**Comment by SvanteRichter** [comment]
A fix was merged as you can see above. Are you using a version that includes that?
